src/version.rs

```diff
@@ -565,6 +567,14 @@ mod tests {
     }
 
     #[test]
+    fn test_display() {
+        let version = Version::parse("1.2.3-rc1").unwrap();
+        assert_eq!(format!("{:20}", version), "1.2.3-rc1           ");
```

Comment:
The test fails here. I don't understand why.

Reply:
What's the actual output you're getting here?

Reply:
```
---- version::tests::test_display stdout ----
	thread 'version::tests::test_display' panicked at 'assertion failed: `(left == right)`
  left: `"1.2.3-rc1"`,
 right: `"1.2.3-rc1           "`', src/version.rs:572:8
```

Reply:
Oh right. So the format isn't respecting the padding length... I quickly tried to run the code today but Windows said no to compiling `libgit-sys`.